tests/translator/input/connector_api_to_multiple_function.yaml

```diff
@@ -0,0 +1,52 @@
+Resources:
```

Comment:
We can now add tests that combine all 4 possible combinations from https://github.com/aws/serverless-application-model/discussions/2761#discussioncomment-4660695.

Reply:
Added a test that use embedded connector with multiple destinations. So we have all 4 combinations in our tranform tests.